CMakeLists.txt

```diff
@@ -31,7 +46,18 @@ cuda_add_library(gdf SHARED
     src/joining.cu
     src/scan.cu
     src/segmented_sorting.cu
+    src/filterops.cu
+    src/bitmaskops.cu
+	src/streamcompactionops.cu
+    src/datetimeops.cu
 )
+if (NOT arrow)
```

Comment:
With the setup using the conda environment, arrow is already included and available.  This check is not able to find the arrow build in conda.  Commenting out this block will "fix" it though.